### PR TITLE
Make Decimal fields null if no value supplied

### LINKED ISSUE #2672: productVariantCreate mutation fails
Mutation `productVariantCreate` fails in the new dashboard when creating a variant without "cost price" or "price override" values.

Mutation variables that resulted in an error:

```
{
  "attributes": [
    {
      "slug": "cover",
      "value": "soft"
    }
  ],
  "costPrice": "",
  "priceOverride": "",
  "product": "UHJvZHVjdDo2Mg==",
  "quantity": "5",
  "sku": "cover-soft",
  "trackInventory": true
}
```

Error:
```
{"errors":[{"message":"Variable \"$costPrice\" got invalid value \"\".\nExpected type \"Decimal\", found \"\".","locations":[{"line":103,"column":60}]}]}
```

The problem is that the fields that failed are of `Decimal` type which doesn't accept an empty string `""`  value. I ran the mutation manually and set `null` instead of empty strings and it worked.

### What I'm trying to achieve
Create a product variant in the new dashboard.

### Steps to reproduce the problem
1. Create a new product.
2. Try to add a new variant with empty "cost price" and "price override" fields.

### What I expected to happen
A product variant should be added.

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it fixes bug causing error when no `costPrice` or `priceOverride` were provided.
Fixes #2672 

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).

### CODE REVIEW

**Comment by maarcingebala** [comment]
Do we have any other occurrences of `Decimal` variables? I'd make sure that we cover all of them.

File: saleor/static/dashboard-next/products/views/ProductVariantCreate.tsx [hunk 1]
```diff
@@ -79,8 +79,12 @@ export const ProductVariant: React.StatelessComponent<ProductUpdateProps> = ({
                       variantCreate({
                         variables: {
                           attributes: formData.attributes,
-                          costPrice: formData.costPrice,
-                          priceOverride: formData.priceOverride,
+                          costPrice: formData.costPrice
```
Comment (patrys) [inline saleor/static/dashboard-next/products/views/ProductVariantCreate.tsx]: What was submitted before? Shouldn't there be a helper function to do it?

Comment (dominik-zeglen) [reply saleor/static/dashboard-next/products/views/ProductVariantCreate.tsx]: An empty string. It seems that we need to pass either string (which cannot be empty) or null. The problem is that cannot solve this problem by using just TS types, unless we pass `--passthrough-custom-scalars` option to `apollo-codegen` and write constructor for all those custom types.

**Comment by dominik-zeglen** [comment]
Now all decimals had been covered.

**Pull request merged**